### PR TITLE
(DOCSP-11398): Update Telephone Numbers page

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-11398)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11398/style-guide/style/telephone-numbers.html)

### CODE REVIEW

File: source/style-guide/style/telephone-numbers.txt [hunk 1]
```diff
@@ -4,8 +4,6 @@
 Telephone Numbers
 =================
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 Use these guidelines when showing telephone numbers in content.
```
Comment (atsansone) [inline source/style-guide/style/telephone-numbers.txt:7]:
```suggestion
.. default-domain:: mongodb

.. include:: /includes/styles/corrections.rst

Use these guidelines when showing telephone numbers in content.
```

File: source/style-guide/style/telephone-numbers.txt [hunk 2]
```diff
@@ -22,7 +20,7 @@ number.
    * - 1 800 961 4454
      - 1 (800) 961-4454
 
-Precede US and Canadian telephone numbers with ``+1``. Precede all
+Precede US and Canadian telephone numbers with ``1``. Precede all
 other telephone numbers with a plus sign.
```
Comment (atsansone) [inline source/style-guide/style/telephone-numbers.txt:24]: **issue:** This is important: Non-US or Canadian users would use `+1` and many of our customers are outside of North America.

[Twilio](https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers), a company that lives off phone numbers, uses the [E.164 standard](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-E.164-201011-I!!PDF-E&type=items). Twilio [explains the format](https://www.twilio.com/docs/glossary/what-e164).

```suggestion
Precede US and Canadian telephone numbers with ``+1``. Precede all
other telephone numbers with a plus sign.
```

Comment (melissamahoney-mongodb) [reply source/style-guide/style/telephone-numbers.txt:24]: I definitely misunderstood what this text was getting at, and the example following it is wrong too. Updated everything.

File: source/style-guide/style/telephone-numbers.txt (hunk 1 above)
Comment (melissamahoney-mongodb) [reply source/style-guide/style/telephone-numbers.txt:11]: Already added.

**Pull request merged**